fineract-provider/src/main/resources/db/changelog/tenant/upgrades/0000_upgrade_to_1.5.xml

```diff
@@ -0,0 +1,100 @@
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied. See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <!-- V362__reschedule_interest_to_lrs.sql -->
+    <changeSet author="fineract" id="362">
+        <addColumn tableName="m_loan_repayment_schedule">
+            <column name="reschedule_interest_portion" type="decimal(19,6)" defaultValueComputed="NULL" position="14" />
+        </addColumn>
+    </changeSet>
+    <!-- V363__mnote_indexing_for_performance.sql -->
+    <changeSet author="fineract" id="363">
+        <createIndex indexName="savings_account_transaction_id" tableName="m_note">
+            <column defaultValueComputed="NULL" name="savings_account_transaction_id"/>
+        </createIndex>
+    </changeSet>
+    <!-- V364__elastic_hook_template.sql -->
+    <changeSet author="fineract" id="364">
+        <insert tableName="m_hook_templates">
+            <column name="id" valueNumeric="3"/>
+            <column name="name" value="Elastic Search"/>
+        </insert>
+        <insert tableName="m_hook_schema">
+            <column name="id" valueNumeric="8"/>
+            <column name="hook_template_id" valueNumeric="3"/>
+            <column name="field_type" value="string"/>
+            <column name="field_name" value="Payload URL"/>
+            <column name="placeholder" value="http://&lt;host&gt;/&lt;index name&gt;/&lt;type name&gt;"/>
+            <column name="optional" valueBoolean="false"/>
+        </insert>
+        <insert tableName="m_hook_schema">
+            <column name="id" valueNumeric="9"/>
+            <column name="hook_template_id" valueNumeric="3"/>
+            <column name="field_type" value="string"/>
+            <column name="field_name" value="Content Type"/>
+            <column name="placeholder" value="json"/>
+            <column name="optional" valueBoolean="false"/>
+        </insert>
+        <insert tableName="m_hook_schema">
+            <column name="id" valueNumeric="10"/>
+            <column name="hook_template_id" valueNumeric="3"/>
+            <column name="field_type" value="string"/>
+            <column name="field_name" value="Index Name"/>
+            <column name="placeholder"/>
+            <column name="optional" valueBoolean="true"/>
+        </insert>
+    </changeSet>
+    <!-- V365__reportCategoryList-FINERACT-1306.sql -->
+    <changeSet author="fineract" id="365">
```

Comment:
V365__reportCategoryList-FINERACT-1306.sql looks different in flyway:
the two tables are not missing?

Reply:
Same here... maybe @josehernandezfintecheandomx can shed some light?